packages/ag-charts-community/src/chart/themes/chartTheme.ts

```diff
@@ -455,7 +455,13 @@ export class ChartTheme {
                     highlightStyle: { fill: { $ref: 'foregroundColor' } },
                     label: { color: { $ref: 'textColor' } },
                 },
-                fill: { $ref: 'chartBackgroundColor' },
+                fill: {
+                    $if: [
+                        { $eq: [{ $path: './position/floating' }, true] },
+                        { $ref: 'chartBackgroundColor' },
+                        'transparent',
+                    ],
```

Comment:
I think this should work and save an operation (the 2nd item in `$path` is a fallback value if the path is not found):

```suggestion
                    $if: [
                        { $path: ['./position/floating', false] },
                        { $ref: 'chartBackgroundColor' },
                        'transparent',
                    ],
```